offload/include/Shared/Types.h

```diff
@@ -1,22 +0,0 @@
-//===-- Shared/Types.h - Type defs shared between host and device - C++ -*-===//
-//
-// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
-// See https://llvm.org/LICENSE.txt for license information.
-// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-//
-//===----------------------------------------------------------------------===//
-//
-// Environments shared between host and device.
-//
-//===----------------------------------------------------------------------===//
-
-#ifndef OMPTARGET_SHARED_TYPES_H
-#define OMPTARGET_SHARED_TYPES_H
-
-#ifndef OMPTARGET_DEVICE_RUNTIME
```

Comment:
Can this `OMPTARGET_DEVICE_RUNTIME` definition go as well?

Reply:
The whole file is deleted.

Reply:
Yeah, that's what I wanted to ask.